src/external/VTK.jl

```diff
@@ -0,0 +1,58 @@
+import WriteVTK: vtk_grid, vtk_save
+
+# Uses WriteVTK.jl to convert scfres structure to VTk file format
+
+"""
+    save_scfres(filename, scfres, Val(:vtk))
+
+The function takes in the VTK filename and the scfres structure and stores into a VTK file.
+
+Grid Values:
+
+- ρ -> Density in real space
+- ψreal\\_i\\_j -> Real values of Bloch waves in real space, where i and j are Kpoint and EigenVector indexes respectively
```

Comment:
Line too long: We try to keep lines below 90 char (but we are not super strict about it, so a few chars more are ok.)

Reply:
I am going with with `band` and `k` so i don't think I will need such a long docstring anyway.